recipe/meta.yaml

```
@@ -1,31 +1,32 @@
 {% set name = "opentelemetry-propagator-b3" %}
-{% set version = "1.28.2" %}
+{% set version = "1.26.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opentelemetry_propagator_b3-{{ version }}.tar.gz
-  sha256: 0e0b204e8eee3042c0d9fb3cd6f1a75b2a4af9b0a539a18ad510b453d563f876
+  sha256: aa05f951099f7b402139de64aa9169dc692e02741ee95ad70a1454a6d4c76c81
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script:  {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
+  skip: True  # [py<38]
   number: 0
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - hatchling
   run:
-    - python >=3.7
+    - python
     - deprecated >=1.2.6
     - opentelemetry-api ~=1.3
```

```suggestion
    - opentelemetry-api >=1.3,<1.4
```
I wonder if `conda-build` is able to parse `~=`

---

It does! https://docs.conda.io/projects/conda-build/en/stable/resources/package-spec.html#package-match-specifications